src/server/routes/embed.js

```diff
@@ -0,0 +1,29 @@
+/* @flow */
+
+import type { $Request, $Response } from 'express'
+
+module.exports = (req: $Request, res: $Response) => {
+  const { id } = req.params
+  const title = 'Great title'
+  const description = 'Cool video, please watch it.'
+  const thumbnailUrl = 'http://paratii.video/imagens/cropped-logo_colorido_horizontal.png'
+  const url = `https://portal.paratii.video/play/${id}`
+
+  res.send(`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>${title}</title>
+        <link rel="stylesheet" type="text/css" href="/embed/index.css">
+        <meta name="og:title" content="${title}"/>
```

Comment:
Here is where we set our `html` headers. The rest of the embed content is rendered on the FE